tests/unit/test_config.py

```diff
@@ -5,7 +5,25 @@
 from linker.configuration import Config
 from tests.unit.conftest import ENV_CONFIG_DICT, PIPELINE_CONFIG_DICT
 
-# TODO [MIC-4491]: beef these up
+
+def test_config_instantiation(test_dir, config):
+    assert config.pipeline == PIPELINE_CONFIG_DICT["good"]
+    assert config.environment == ENV_CONFIG_DICT
+    assert config.input_data == [
+        Path(x) for x in [f"{test_dir}/input_data{n}/file{n}" for n in [1, 2]]
+    ]
```

Comment:
Is this testing all the necessary config files are provided? Hard to understand exactly what the list comprehension is

Reply:
It's just checking that config.input_data is a list of the expected input datasets (which in this test case are <test-dir>/input_data{1,2}/file{1,2}).

Reply:
And that is the expected list of input files because that is what the "good_input_data.yaml" in the conftest.py says they are.